gradle/libs.versions.toml

```diff
@@ -19,7 +19,7 @@ androidx-constraintlayout-compose = '1.0.1'
 androidx-core = '1.13.1'
 androidx-fragment = '1.8.4'
 androidx-lifecycle = '2.8.5'
-androidx-navigation = '2.7.7'
+androidx-navigation = '2.8.3'
```

Comment:
There's a bug in 2.8.1 (used by QuickEditor) that causes some Lint warnings. WP is configured to fail on those so I had to bump the library here to fix the issue. 

Here's the issue tracker with a bit more info - https://issuetracker.google.com/issues/368070326

We will update the library in the Gravatar SDK as well, but for now, this should do the trick. 

@nbradbury Any thoughts on this?

Reply:
Normally I'd opt for updating a library in a dedicated PR, but I just tested the app separately with androidx-navigation v2.8.3 and ran into no problems, so I'm fine with updating it here.